
2018-06-11 16:38:47,585 INFO     127.0.0.1 D:\ergou\EgouBlog\blog\views.py[line:63] 获取了全部用户列表
2018-06-11 16:39:07,390 INFO     127.0.0.1 D:\ergou\EgouBlog\blog\views.py[line:28] 获取主页设置
2018-06-11 17:12:57,790 INFO     127.0.0.1 D:\ergou\EgouBlog\blog\views.py[line:28] 获取主页设置
2018-06-12 16:35:51,867 ERROR    127.0.0.1 D:\ergou\EgouBlog\blog\views.py[line:458] 文件上传成功 Huanghei
2018-06-12 16:35:51,885 ERROR    D:\ergou\EgouBlog\blog\views.py[line:466] 文件保存错误 [Errno 2] No such file or directory: 'D:\\ergou\\EgouBlog\\static\\file_temp\\Sj3jGIEMOClmPy3i9AuwoFhXob0r.jpg\\3ond4WUmQBpb15sjK5dcICqOIwPw.jpg'
2018-06-12 16:36:57,055 ERROR    127.0.0.1 D:\ergou\EgouBlog\blog\views.py[line:458] 文件上传成功 Huanghei
2018-06-12 16:36:57,056 ERROR    D:\ergou\EgouBlog\blog\views.py[line:466] 文件保存错误 [Errno 2] No such file or directory: 'D:\\ergou\\EgouBlog\\static\\file_temp\\r2ivMvoQIr5cOCKsNAiLFVq5XBx6.jpg\\C2XsCltHlzqvXDS9BJJiGDCKKp8K.jpg'
2018-06-12 16:37:21,320 ERROR    127.0.0.1 D:\ergou\EgouBlog\blog\views.py[line:458] 文件上传成功 Huanghei
2018-06-12 16:37:21,321 ERROR    D:\ergou\EgouBlog\blog\views.py[line:466] 文件保存错误 [Errno 2] No such file or directory: 'D:\\ergou\\EgouBlog\\static\\file_temp\\BtP1ak56l8NmMctpt3Sy3oH4mgBn.jpg\\OgaqzvhIaZCmhr52qKGcgbX24uci.jpg'
2018-06-12 16:38:37,963 ERROR    127.0.0.1 D:\ergou\EgouBlog\blog\views.py[line:458] 文件上传成功 Huanghei
2018-06-12 16:38:52,782 ERROR    127.0.0.1 D:\ergou\EgouBlog\blog\views.py[line:458] 文件上传成功 Huanghei
2018-06-12 17:56:37,990 ERROR    D:\ergou\EgouBlog\blog\dataHelper.py[line:161] 更新数据错误！错误信息:'Site' object has no attribute 'update'
2018-06-12 17:57:46,578 INFO     D:\ergou\EgouBlog\blog\dataHelper.py[line:167] 更新数据成功
2018-06-12 17:58:18,156 INFO     D:\ergou\EgouBlog\blog\dataHelper.py[line:167] 更新数据成功
2018-06-13 00:55:30,025 INFO     127.0.0.1 D:\ergou\ErgouBlog\blog\views.py[line:52] 用户登录成功 USER_NAME:Huanghei
2018-06-13 00:56:23,000 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:193] 更新数据错误！错误信息:'QuerySet' object has no attribute 'user_site'
2018-06-13 00:57:51,539 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:193] 更新数据错误！错误信息:'QuerySet' object has no attribute 'user_site'
2018-06-13 00:58:13,661 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:199] 更新数据成功
2018-06-13 00:58:30,971 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:193] 更新数据错误！错误信息:'UserSite' object has no attribute 'update'
2018-06-13 00:58:40,505 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:193] 更新数据错误！错误信息:Manager isn't accessible via UserSite instances
2018-06-13 00:59:16,469 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:199] 更新数据成功
2018-06-13 01:03:58,054 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:180] 查询用户错误！错误信息:type object 'UserSite' has no attribute 'User_set'
2018-06-13 01:04:10,223 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:180] 查询用户错误！错误信息:'Manager' object has no attribute 'User_set'
2018-06-13 01:04:32,845 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:180] 查询用户错误！错误信息:'QuerySet' object has no attribute 'User_set'
2018-06-13 01:10:30,164 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:180] 查询用户错误！错误信息:'Manager' object has no attribute 'filer'
2018-06-13 01:10:42,476 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:180] 查询用户错误！错误信息:'Manager' object has no attribute 'filer'
2018-06-13 01:12:34,995 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:193] 更新数据错误！错误信息:'QuerySet' object has no attribute 'user_site'
2018-06-13 01:12:49,143 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:180] 查询用户错误！错误信息:Cannot resolve keyword 'user_user_name' into field. Choices are: blog_bgm, blog_head_color, blog_info, blog_name, id, user
2018-06-13 01:13:14,830 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:180] 查询用户错误！错误信息:invalid literal for int() with base 10: 'Huanghei'
2018-06-13 01:13:48,554 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:193] 更新数据错误！错误信息:'QuerySet' object has no attribute 'user_site'
2018-06-13 01:14:17,688 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:199] 更新数据成功
2018-06-13 01:15:51,872 ERROR    127.0.0.1 D:\ergou\ErgouBlog\blog\views.py[line:567] 信息POST不完整
2018-06-13 01:16:08,867 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:167] 更新数据成功
2018-06-13 01:32:19,152 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:229] 更新数据成功
2018-06-13 01:33:47,236 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:229] 更新数据成功
2018-06-13 01:33:57,675 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:229] 更新数据成功
2018-06-13 21:19:56,382 ERROR    127.0.0.1 D:\ergou\ErgouBlog\blog\views.py[line:60] 用户名或密码错误
2018-06-13 21:20:07,957 INFO     127.0.0.1 D:\ergou\ErgouBlog\blog\views.py[line:52] 用户登录成功 USER_NAME:Huanghei
2018-06-13 21:20:11,359 INFO     127.0.0.1 D:\ergou\ErgouBlog\blog\views.py[line:52] 用户登录成功 USER_NAME:Huanghei
2018-06-13 21:25:29,574 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:301] 更新数据错误！错误信息:'Manager' object has no attribute 'filer'
2018-06-13 21:30:11,142 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:301] 更新数据错误！错误信息:'Manager' object has no attribute 'filer'
2018-06-13 21:30:36,038 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:264] 更新成功
2018-06-13 21:31:29,713 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:264] 更新成功
2018-06-13 21:31:56,693 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:264] 更新成功
2018-06-13 21:32:19,925 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:255] 添加成功
2018-06-13 21:32:51,671 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:285] 分类中还有文章无法删除！
2018-06-13 21:33:17,402 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:301] 更新数据错误！错误信息:ArticleClass matching query does not exist.
2018-06-13 21:33:25,753 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:301] 更新数据错误！错误信息:ArticleClass matching query does not exist.
2018-06-13 21:33:32,829 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:301] 更新数据错误！错误信息:ArticleClass matching query does not exist.
2018-06-13 21:33:37,942 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:285] 分类中还有文章无法删除！
2018-06-13 21:34:36,499 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:285] 分类中还有文章无法删除！
2018-06-13 21:35:31,084 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:286] 分类中还有文章无法删除！
2018-06-13 21:36:05,016 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:279] 删除成功！
2018-06-13 21:36:14,347 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:286] 分类中还有文章无法删除！
2018-06-13 21:38:12,596 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:261] 添加失败
2018-06-13 21:38:27,013 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:256] 添加成功
2018-06-13 21:38:44,678 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:261] 添加失败
2018-06-13 21:39:38,181 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:311] 更新数据错误！错误信息:The QuerySet value for an exact lookup must be limited to one result using slicing.
2018-06-13 21:44:40,728 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:311] 更新数据错误！错误信息:Cannot resolve keyword 'set_user_name' into field. Choices are: Article, User, class_create_time, class_name, id
2018-06-13 21:45:10,064 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:311] 更新数据错误！错误信息:The QuerySet value for an exact lookup must be limited to one result using slicing.
2018-06-13 21:46:29,879 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:261] 添加失败
2018-06-13 21:46:50,335 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:256] 添加成功
2018-06-13 21:47:48,998 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:273] 更新成功
2018-06-13 22:23:10,876 ERROR    127.0.0.1 D:\ergou\ErgouBlog\blog\views.py[line:755] 信息POST不完整
2018-06-13 22:27:46,724 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:423] 更新数据错误！错误信息:local variable 'class_obj' referenced before assignment
2018-06-13 22:28:13,602 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:346] 无法获取文章分类 错误信息'class_id'
2018-06-13 22:28:28,625 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:346] 无法获取文章分类 错误信息'class_id'
2018-06-13 22:30:04,971 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:361] 添加文章成功
2018-06-13 22:32:50,847 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:386] 更新成功
2018-06-13 22:33:09,486 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:406] 无法获取文章！
2018-06-13 22:37:30,568 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:406] 无法获取文章！
2018-06-13 22:39:50,726 ERROR    D:\ergou\ErgouBlog\blog\dataHelper.py[line:406] 无法获取文章！
2018-06-13 22:40:11,688 INFO     D:\ergou\ErgouBlog\blog\dataHelper.py[line:399] 删除成功！
2018-06-14 16:03:39,392 ERROR    D:\ergou\EgouBlog\blog\dataHelper.py[line:440] 无法获取文章！
2018-06-14 16:04:29,022 ERROR    D:\ergou\EgouBlog\blog\dataHelper.py[line:425] 文章移动分类失败
2018-06-14 16:04:46,483 ERROR    D:\ergou\EgouBlog\blog\dataHelper.py[line:425] 文章移动分类失败
2018-06-14 16:05:15,893 ERROR    D:\ergou\EgouBlog\blog\dataHelper.py[line:425] 文章移动分类失败
2018-06-14 16:07:02,263 ERROR    D:\ergou\EgouBlog\blog\dataHelper.py[line:425] 文章移动分类失败 错误:Article has no field named 'user_article_class'
2018-06-14 16:07:37,578 INFO     D:\ergou\EgouBlog\blog\dataHelper.py[line:433] 文章移动分类成功
2018-06-14 16:07:53,593 INFO     D:\ergou\EgouBlog\blog\dataHelper.py[line:433] 文章移动分类成功
